datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -271,6 +271,20 @@ def create_scraper_configuration(self, instance=None):
 
         config['telemetry'] = is_affirmative(instance.get('telemetry', default_instance.get('telemetry', False)))
 
+        # The metric name services use to indicate build information
+        config['metadata_metric_name'] = instance.get(
+            'metadata_metric_name', default_instance.get('metadata_metric_name')
+        )
+
+        # Map of metadata key names to label names
+        config['metadata_label_map'] = instance.get(
+            'metadata_label_map', default_instance.get('metadata_label_map', {})
+        )
+
+        config['_default_metric_transformers'] = {}
+        if config['metadata_metric_name'] and config['metadata_label_map']:
+            config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
```

Comment:
what's the behavior if we want to have a custom procesing on the `metadata_metric_name` along the `transform_metadata` function, we only have one map right? 🤔

Reply:
You can override the `transform_metadata` method